startup.sh

```diff
@@ -9,13 +9,19 @@ function waitForDB() {
   DB_USER=$(awk -F '[/:@]' '{print $4}' <<< $DATABASE_URL)
   DB_PASS=$(awk -F '[/:@]' '{print $5}' <<< $DATABASE_URL)
   DB_HOST=$(awk -F '[/:@]' '{print $6}' <<< $DATABASE_URL)
+  DB_PORT=$(awk -F '[/:@]' '{print $7}' <<< $DATABASE_URL)
```

Comment:
what happens if there is no `$7`?

Reply:
the script test the var DB_PORT and if it's NaN we will use default port